packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.Props.ts

```diff
@@ -48,6 +48,12 @@ export interface IContextualMenuProps extends React.Props<ContextualMenu>, IWith
   directionalHint?: DirectionalHint;
 
   /**
+   * How the element should be positioned in RTL layouts.
+   * If not specified, a mirror of `directionalHint` will be used instead
+   */
+  directionalHintForRtl?: DirectionalHint;
```

Comment:
maybe we should just name this `overrideRTLFlip` or `forceLTRDirection` ? Having a hard time understanding `directionalHintForRtl`